src/shared/services/map-location/index.js

```diff
@@ -19,6 +24,20 @@ export const address2pdok = address => {
   };
 };
 
+export const wktPointToLocation = wktPoint => {
+  if (!wktPoint.includes('POINT')) {
+    throw TypeError('Provided WKT geometry is not a point.');
+  }
+  const coordinate = wktPoint.split('(')[1].split(')')[0];
+  const lat = parseFloat(coordinate.split(' ')[1]);
+  const lng = parseFloat(coordinate.split(' ')[0]);
+
+  return {
+    lat,
+    lng,
+  };
+};
+
```

Comment:
This function doesn't seem to have a test case. Also, the implementation expects a specific object as parameter. Provide a docblock that described the function as well as its params and return value.

Reply:
removed